src/components/ComponentList.js

```diff
@@ -128,16 +127,15 @@ export default class ComponentList extends React.Component {
   }
 
   render() {
-    const { loadMoreRows, listHeight, noRowsRenderer, list, listLength, renderFilterBar } = this.props
+    const { loadMoreRows, noRowsRenderer, list, listLength, renderFilterBar } = this.props
     const { sortOrder, contentSeq } = this.state
     return (
-      <div>
+      <div style={{ flex: '1' }}>
```

Comment:
Can we put this in a css class? Maybe something reusable all over the site where needed